test/fuzz_rdflib.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import atheris
+
+with atheris.instrument_imports():
+    from rdflib import Graph
+    import sys
+    from xml.sax import SAXParseException
+
+
+def TestOneInput(data):
```

Comment:
I know this is the style that atheris examples use, but I would prefer rather using PEP-8 compatible naming.

```suggestion
def test_one_input(data):
```

Reply:
Also, what is happening will be clearer to reviewers if there are type hints, I am not sure what data will be here, and not sure why you check the length of data and why it must be less than 20.

Reply:
Done. Let me know if anything needs more clarification.